projects/iot-cloud/software/lora2mqtt/src/main.py

```diff
@@ -0,0 +1,147 @@
+import json
+import logging
+import os
+import sys
+import atexit
+
+import paho.mqtt.publish as publish
+import requests
+from apscheduler.schedulers.background import BackgroundScheduler
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flatten_json import flatten
+
+from utils.exceptions import RestException
+from utils.functions import resolve_mapping
+
+application = Flask(__name__)
+
+CONFIG_URL = os.environ.get('CONFIG_URL',
+                            'https://gist.githubusercontent.com/dorinelfilip/6743182de240959b2a723fcf232b8496/raw/'
+                            'f37c2be3e3e5f6abc54b16f14b7643ce0ab39d42/lora2mqtt.json')
+
+MQTT_HOST = os.environ.get('MQTT_HOST', 'mqtt.beia-telemetrie.ro')
+MQTT_PORT = os.environ.get('MQTT_PORT', 1883)
+
+
+@application.route('/')
+def hello_world():
+    return jsonify({'result': 'hello world'})
+
+
+@application.route('/publish', methods=['POST'])
+def publish_data():
+    data = request.get_json(silent=True)
```

Comment:
Shouldn't you check if you actually have a JSON payload?

Reply:
That is exactly what I do in the next line (`if not data`). It will return None if there is no valid JSON payload.

The only corner case that I miss is if you give an empty dict `{}` as a payload, because it is also a False value, but I think that is not so important.